WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -103,7 +103,7 @@ public static int Main(string[] args)
 
 					Logger.LogError(ex);
 
-					RxApp.MainThreadScheduler.Schedule(() => throw ex);
+					RxApp.MainThreadScheduler.Schedule(() => throw new ApplicationException("Command execution crashed", ex));
```

Comment:
Is it always a command that crashed?

Reply:
No. Good catch :)
It will happen 98% of times for a command, but it doesn't need to.